libs/wingsdk/src/index.ts

```diff
@@ -1,11 +1,6 @@
+// this file must only export the standard library types and not target-specific types.
 export * as cloud from "./cloud";
 export * as core from "./core";
 export * as redis from "./redis";
```

Comment:
Not a blocker for this PR, but redis probably shouldn't be considered part of our "standard library". It's a specific implementation.

Reply:
We intend to extract it completely to a 3rd party library in the future but for now we decided that it will be shipped with the SDK under the `redis` submodule.